### PR TITLE
Added some basic examples about inheritance

### PR DESCRIPTION
**Pull request by mariomac**
Please don't hesitate to ask me for any change in the format/structure of the document.

Signed-off-by: Mario Macias <mariomac@gmail.com>

### CODE REVIEW

**Comment by hhariri** [comment]
This is great, thank you. Left some comments inline.

**Comment by mariomac** [comment]
Sorry, I can't see any comment. Anyway I'll resubmit another patch with some typos/missings.

File: docs-src/inheritance.adoc [hunk 1]
```diff
@@ -0,0 +1,87 @@
+=== Inheritance
+
+==== Basic inheritance
```
Comment (mariomac) [inline docs-src/inheritance.adoc]: Added a "namedArguments.adoc" file.

File: docs-src/inheritance.adoc [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+fun main(args: Array<String>) {
+    val dog: Dog = Poodle("Rufo")
+    dog.sayHello()
+}
+// Rufo, the dog from France says: wow wow!
+----
+<1> Observe that `name` is neither a `var` nor `val` because it is a
+    constructor argument for `Yorkshire`, whose value is passed to the `name`
+    attribute of `Dog` (it does not override it).
+<2> Named arguments allow clearer invocations and avoid runtime errors (e.g.
```
Comment (mariomac) [inline docs-src/inheritance.adoc]: Right, I had it in mind. I'm a bit chaotic when I build up things :-) Added a namedArguments.adoc section.

**Pull request merged**